test/contrib/rack/helpers.rb

```diff
@@ -1,9 +1,12 @@
 require 'helper'
+require 'net/http'
 require 'ddtrace'
 require 'ddtrace/contrib/rack/middlewares'
 
 require 'rack/test'
 
+Datadog::Monkey.patch_module(:http)
```

Comment:
Yes I see. I don't want to spend a lot of time on it, neither duplicate our testing code. Let's keep it that way, also because having the `unpatch()` method for our integrations, will simplify this task a lot.

But it's another thing that isn't required in this PR.